constellations/gemini-admin/src/account/accountEdit.js

```diff
@@ -0,0 +1,204 @@
+"use strict";
+require("./accountEdit.sass");
+import React from "react";
+import API from "../api.js";
+import { Modal } from "../modals/modal.js";
+import { OkayModal } from "../modals/okayModal.js";
+import { InputText } from "../utils/inputText.js";
+import { setCurrentAccountId } from "../localStorage.js";
+
+export class AccountEditPage extends React.Component {
+    state = {
+        firstName: "",
+        middleName: "",
+        lastName: "",
+        email: "",
+        phone: "",
+        notes: "",
+    };
+
+    handleChange = (event, value) => {
+        this.setState({ [value]: event.target.value });
+    };
+
+    onClickCancel = () => {
+        window.location.hash = "accounts";
+    };
+
+    onClickSave = () => {
+        let account = {
+            primaryEmail: this.state.email,
+            password: "password", // Temporary
+        };
+
+        API.post("api/accounts/create", account)
```

Comment:
Put a TODO here to replace this series of calls with one endpoint when the server has a Create-Account-AND-User endpoint